planner_cspace/test/src/test_start_pose_predictor.cpp

```diff
@@ -68,7 +68,7 @@ class StartPosePredictorTester : public ::testing::Test
     map_info_.origin.orientation.w = 1.0;
     map_info_.linear_resolution = 1.0;
     map_info_.angular_resolution = 2 * M_PI / map_info_.angle;
-    interpolator_.reset(map_info_.linear_resolution, 1);
```

Comment:
The previous test was wrong:
`interpolator_.reset(map_info_.linear_resolution, 1);`
should be
`interpolator_.reset(map_info_.angular_resolution, 1);`

It is needed to change the `SwitchBack` test slightly with the correct interpolations.